src/api/app/controllers/webui/projects/maintenance_incidents_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Webui
+  module Projects
+    class MaintenanceIncidentsController < WebuiController
+      before_action :set_project, only: [:index, :create_request]
+      before_action :lockout_spiders, only: [:index]
+      before_action :require_login, only: [:create, :create_request]
+
+      after_action :verify_authorized, except: [:index, :create, :create_request]
```

Comment:
We should verify this for `:create`

Reply:
I realized, it should also be the case for `create_request`. Login is required, but then we never check authorization.